src/core-layers/path-layer/path-layer-vertex.glsl.js

```diff
@@ -68,8 +68,10 @@ vec3 lineJoin(vec3 prevPoint, vec3 currPoint, vec3 nextPoint) {
 
   float lenA = length(deltaA);
   float lenB = length(deltaB);
-  vec2 dirA = lenA > 0. ? normalize(deltaA) : vec2(1.0, 0.0);
-  vec2 dirB = lenB > 0. ? normalize(deltaB) : vec2(1.0, 0.0);
+  lenA = lenA > 0.1 ? lenA : 0.0;
```

Comment:
* Would be great to add comments explaining why we do this.
* Should we use EPSILON instead of 0.1 (or if we need a separate value, maybe create a new constant?

Reply:
Please use EPSILON

Reply:
I will add more comments, can't use EPSILON here cause the unit here is pixel.